src/components/List/List.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export const UserList = (props) => {
+  return (
+    <li onClick={props.onClick}></li>
+  );
+};
+  
+UserList.props = {
```

Comment:
Why if you have Typescript your props are here as prototype and isn't a interface ? Check the button component